src/sql/SelectStatement.h

```diff
@@ -58,7 +75,19 @@ namespace hsql {
     Expr* whereClause;
     GroupByDescription* groupBy;
 
-    SelectStatement* unionSelect;
+    // Note that a SetOperator is always connected to a
+    // different SelectStatement. This statement can itself
+    // have SetOperator connections to other SelectStatements.
+    // To evaluate the operators in the correct order: 
+    //    1. Fully evaluate the nested SelectStatement
+    //    2. Connect the nested SelectStatement to our original
+    //       with the given SetOperation
+    //    3. Connect the result with the next SetOperator in the
+    //       vector
+    //    4. Repeat until all SetOperators in the vector are
+    //       connected 
```

Comment:
I‘m not sure if someone who didn‘t work through this knows what you mean with „connect“. Also, what is „our original“? Maybe give an example and tell people how to understand it? I‘d also include the outer order/limit.

Reply:
Okay